internal/pkg/vm/internal/vmcontext/validation_test.go

```diff
@@ -0,0 +1,24 @@
+package vmcontext
+
+import (
+	"testing"
+
+	vsuites "github.com/filecoin-project/chain-validation/suites"
+)
+
+func TestChainValidationSuite(t *testing.T) {
+	f := NewFactories()
+
+	vsuites.TestValueTransferSimple(t, f)
+	vsuites.TestValueTransferAdvance(t, f)
+	vsuites.TestAccountActorCreation(t, f)
+
+	// Skipping as this tests uses the payment channel actor code ID and it DNE in go-filecoin
+	// vsuites.TestInitActorSequentialIDAddressCreate(t, f)
+
+	// Skipping since multisig actor DNE in go-filecoin
+	// vsuites.TestMultiSigActor(t, f)
+
+	// Skipping since payment channel actor DNE in go-filecoin
```

Comment:
I don't understand these comments. Please be more specific about what needs to change. Phrase as `TODO xxx when yyy` if possible